src/file.rs

```diff
@@ -209,6 +217,16 @@ impl AsepriteFile {
         &self.tilesets
     }
 
+    /// Construct the image of each tile in the [Tileset].
+    /// The image has width equal to the tile width and height equal to (tile_height * tile_count).
+    pub fn tileset_image(&self, tileset_id: &TilesetId) -> Option<RgbaImage> {
```

Comment:
I think this should return a result so that we can return an error message in case of failure.
Afaict, the only error case that isn't covered by validation is an invalid tileset id?

Reply:
Right, since this function is exposed to the end-user, they could pass in any unknown tileset id.